src/core/lib/hh/date.cc

```diff
@@ -104,12 +104,18 @@ int last_day_of_month(int y, int m) noexcept {
 }
 
 
-// Returns day of week in civil calendar [0, 6] -> [Sun, Sat]
+// Returns day of week in civil calendar
+//   iso:    [1 .. 7] <-> [Mon, Tue, Wed, Thu, Fri, Sat, Sun]
+//   bible:  [1 .. 7] <-> [Sun, Mon, Tue, Wed, Thu, Fri, Sat]
+//
 // Preconditions:
 //   z is number of days since 1970-01-01 and is in the range:
 //   [numeric_limits<int>::min(), numeric_limits<int>::max()-4].
-int weekday_from_days(int z) noexcept {
-  return (z >= -4) ? (z+4) % 7 : (z+5) % 7 + 6;
+int iso_weekday_from_days(int z) noexcept {
+  return (z >= -3) ? (z+3) % 7 + 1 : (z+4) % 7 + 7;
+}
+int bible_weekday_from_days(int z) noexcept {
+  return (z >= -4) ? (z+4) % 7 + 1 : (z+5) % 7 + 7;
 }
```

Comment:
Are you planning to use this function at some point?

Reply:
We might add it, as an alternative. I'm not sure at this point.